models/solutions.js

```diff
@@ -86,6 +86,8 @@ module.exports = {
         default : "Domains",
         type : String
       },
-      criteriaLevelReport : Boolean
+      criteriaLevelReport : Boolean,
+      license:Object,
+      creatorName:String
```

Comment:
@rakeshSgr what is the difference in value between author and creator? Why did we add a new creatorName?